client/lib/user/shared-utils.js

```diff
@@ -6,8 +6,15 @@ import config from '@automattic/calypso-config';
 /**
  * Internal dependencies
  */
+import wpcom from 'calypso/lib/wp';
```

Comment:
@tyxla : Unsolicited code review alert 😄

Since this "shared utils" modulenow hosts several entirely separate things, one of which imports the large `wpcom` library, it may make sense to split it into multiple files inside a `shared-utils` folder, to ensure tree-shakeability through modularity. I realise that loading a user always happens on boot today, bu who knows, maybe that won't be the case in the future 🙂 There are logged-out pages, after all.

Reply:
That's a good point, and something I've been considering anyway 😅  But thanks for bringing it up! It's a good improvement to follow up with, in any case.

Reply:
Oh, and the unsolicited code reviews are most welcome 🙇‍♂️

Reply:
> Oh, and the unsolicited code reviews are most welcome 🙇‍♂️

Haha, I'm glad that's the case 😄 I don't spend too much time on Calypso these days, but your PR did pique my interest, as the culmination of such a long process in getting rid of the user emitter 🙂 Excellent work, @tyxla! 👍

Reply:
@sgomes I've addressed your suggestion to split the shared utils to modules in #54382.